routes/web.php

```diff
@@ -225,18 +240,18 @@ function () {
             $qrcode = gzencode($qrcode, 9);
         }
         echo $qrcode;
-        
+
         //echo $qrcode->render($data);
-        
+
     }
 );
 Route::get(
-    
+
     'g',
```

Comment:
The first argument in a multi-line function call must be on the line after the opening parenthesis